pkg/libmxnet.pkg

```diff
@@ -21,6 +21,7 @@ It has been built with OpenCV and OpenMP support disabled.
 doc_install_dir = '/usr/share/doc/' + VAR.fullname + '/'
 
 ARGS = FUN.mapfiles('./lib', '/usr/lib', libs, append_suffix=False) + [
-    'LICENSE=' + doc_install_dir + 'copyright',
+    'NOTICE=' + doc_install_dir + 'copyright',
+    'LICENSE=' + doc_install_dir,
```

Comment:
IIUC the `copyright` files are _meant_ to contain both a copyright declaration and the license (in a structured format).

However, it wasn't obvious to me how to readily combine the content of these two upstream files without some hassle (e.g. defining a new make target to generate the `copyright` file), so I made the short-term compromise of putting the copyright notice in the `copyright` file and the license in a separate `LICENSE` file.

It's basically a short term measure to make sure that the initial release of the package will contain full copyright and licensing information.  The idea was to follow up later with a proper solution for a standard `copyright` file.